### PR TITLE
Violet Campos

### PR DESCRIPTION
**Pull request by violetcampos**

### CODE REVIEW

File: 02LessonTwo/homework/index.html [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
-  <link rel="stylesheet" href="css/style.css">
+  <title>First Site</title>
+  <link rel="stylesheet" href="css/style.css" type="text/css">
 </head>
 <body>
 
+    <h1>Goodbye, space!</h1>
+
+    <div>
+      <img src="http://cdn.theatlantic.com/assets/media/img/photo/2015/11/images-from-the-2016-sony-world-pho/s01_130921474920553591/main_900.jpg?1448476701" alt="Jumping Hamster"
```
Comment (fasteddiegarcia) [inline 02LessonTwo/homework/index.html]: Fantastic! Loving the use of the alt tag. It looks like you have a great grasp of the basics. Let's try pushing a little more, like using ALL the tags listed in Homework. Great layout; legible and logical. Nice!

File: 02LessonTwo/in-class-project/index.html [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+
+  <h1>Green Ladies Cleaning</h1>
+
+  <h3>About Us</h3>
+
+    <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English.</p>
+
+  <hr>
+
+  <h3>Services</h3>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/in-class-project/index.html:19]: Where are the Services listed? Maybe a table here? or a UL?

File: 02LessonTwo/homework/index.html [hunk 3]
```diff
@@ -1,26 +1,81 @@
...
+    <a href="http://www.sho.com/sho/homeland/home"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTrQ_PYtS99YNHqv9hGBiEL3IjmCcbrjP_WcZj4CWCuP2XmYq4v7g" alt="Claire Danes, Damien Lewis" title="Homeland" style="height:300px; width:400px"/>
+    </a>
+
+  <h4>Breaking Bad</h4>
+
+    <a href="http://www.amc.com/shows/breaking-bad"><img src="http://unspoiledpodcasts.com/wp-content/uploads/2014/09/Breaking-Bad.jpg" alt="Bryan Cranston, Aaron Paul" title="Breaking Bad" style="height:300px; width:400px"/>
+    </a>
+
+</div>
+
```
Comment (bapplejax) [inline 02LessonTwo/homework/index.html:73]: a great start, Violet! clean, organized code that is almost 100% correctly nested. Remember that all of your content inside the body is nested and is thus in need of a base indentation.

File: 04LessonFour/in-class-project/index.html [hunk 4]
```diff
@@ -1,10 +1,83 @@
...
+    </div>
+
+    <div id="support">
+
+        <h3>Intel Customer Support</h3>
+
+        <p>
+            Find Support Topics by Product, Drivers and Software, or ask the community <a href="#">read more.</a>
+        </p>
+
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html:57]: great work classing and iding your divs! consider using mains and sections to make it a little less confusing later on if you strictly use divs.

File: 05LessonFive/Checkpoint1/css/style.css [hunk 5]
```diff
@@ -1,43 +1,39 @@
...
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.text {
+  width:500px;
+  height:398px;
+  float:right;
```
Comment (bapplejax) [inline 05LessonFive/Checkpoint1/css/style.css:16]: float this guy left as well. we haven't covered floating yet, but once we do it's gonna make sense.

File: 05LessonFive/Checkpoint1/css/style.css [hunk 6]
```diff
@@ -1,43 +1,39 @@
...
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.image img {
+  width:100%;
+  height:100%;
```
Comment (bapplejax) [inline 05LessonFive/Checkpoint1/css/style.css:10]: once you set the width or height equal to 100%, they opposite one will automatically be 100% to constrain those proportions. not wrong here, just redundant.

File: 05LessonFive/Checkpoint1/css/style.css [hunk 7]
```diff
@@ -1,43 +1,39 @@
...
+  float:right;
+  background-color: rgb(240,240,240);
+  text-align: center;
 }
 
-.left h1 {
-    text-align: center;
-}
+a {
+  font-weight: bold;
```
Comment (bapplejax) [inline 05LessonFive/Checkpoint1/css/style.css:22]: font-weight needs numeric values. "bold" values start at 700 and are in- and decrimented in 100's.

File: 07LessonSeven/homework/css/style.css [hunk 8]
```diff
@@ -0,0 +1,57 @@
+body {
+  font-family: arial, sans-serif;
+}
+
+header {
+  font-size: 25px;
+  font-weight: 200;
+  margin-left: 70px;
+}
+
```
Comment (bapplejax) [inline 07LessonSeven/homework/css/style.css:10]: we want the header, nav, and green box to line up vertically, so make them all the same % padding

File: 07LessonSeven/in-class-project/index.html [hunk 9]
```diff
@@ -1,10 +1,72 @@
...
+
+            <h3>Social Media</h3>
+
+            <p>Twitter - @nasa</p>
+            <p>Facebook - fb.com/nasa</p>
+            <p>Instagram - @nasa</p>
+            <p>Youtube - youtube.com/NASA</p>
+
+        </div>
+
```
Comment (bapplejax) [inline 07LessonSeven/in-class-project/index.html:33]: the structure here is great!

File: 07LessonSeven/in-class-project/index.html [hunk 10]
```diff
@@ -1,10 +1,72 @@
...
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Nasa</title>
+
+    <link rel="stylesheet" href="css/my-style.css" type="text.css" />
```
Comment (bapplejax) [inline 07LessonSeven/in-class-project/index.html:7]: if you're wondering why your stylesheet wasn't being applied, you have type="text.css" here when it's text/css  :)

File: 08LessonEight/homework/exercise-06.html [hunk 11]
```diff
@@ -1,10 +1,55 @@
...
+
+<head>
+    <title>Lesson 8 Homework</title>
+
+    <link rel="stylesheet" href="css/exercise-06-style.css" type="text/css">
+
+</head>
+
+<body class="clearfix">
+
```
Comment (bapplejax) [inline 08LessonEight/homework/exercise-06.html:12]: this homework was great! solid structure and appropriate css. good work :D

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+  text-align: center;
+}
+
+header p {
+  margin: 20px 256px;
+  text-align: center;
+}
+
+main {
+  width: 100%;
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/css/homework-style.css:15]: if width is 100% then where is the 1.6% margin going to go? Remember, the whole of the content should be 100% or less

File: 08LessonEight/in-class-project/homework.html [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+
+        <p>Nullam viderer at has, usu an saperet ornatus nominavi. Enim aeque legendos ut mea, his rebum altera te. Ad mel soleat accusam appetere, ad has appareat interpretaris conclusionemque.</p>
+
+    </header>
+
+    <main class="clearfix">
+
+        <section class="main_box">
+
+                <img src="img/keyboard.jpg">
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/homework.html:26]: fix this to be keyboard.JPG in caps. it has to be EXACTLY what's in the name.

File: 08LessonEight/in-class-project/homework.html [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+                    <p>Usu et minim detraxit, mutat sensibus vel ne, meliore dolorem id nam.</p>
+
+                </div>
+
+            </div>
+
+        </aside>
+
+    </main>
+
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/homework.html:75]: your footer is off-center here. and the UL's LIs are indented when they shouldn't. Also, on my screen the last box of the footer boxes goes to the next line.  the positioning is off for the elements. consider have the three main "boxes" being the same width.

**Pull request closed without merging**